.gitignore

```diff
@@ -0,0 +1,40 @@
+*.rbc
```

Comment:
we can use this gitignore from now on since some stuff should be pushed to github